osquery/tables/utility/file.cpp

```diff
@@ -99,6 +127,171 @@ void genFileInfo(const fs::path& path,
     r["type"] = "unknown";
   }
 
+#else
+
+  // Get the handle of the file object.
+  hFile = CreateFile(TEXT(path.string().c_str()),
```

Comment:
nit: As I mentioned above, you could likely remove the declaration of `hFile` altogether, and make this `auto file_handle = CreateFile(...`